### PR TITLE
Update pkg_mgr.php

### PR DESCRIPTION
**Pull request by BBcan177**
* **Note - There are two previously commented out Lines! (L127-128)**

* Mods to if/else/elseif/endif statements for consistency.
* Change package dependency href link icon from "paper-clip" to "globe".
* Put package dependency links into a < ul > for a cleaner output when dependencies span over two lines.
* Add missing gettext()

### CODE REVIEW

File: src/usr/local/www/pkg_mgr.php [hunk 1]
```diff
@@ -179,72 +177,58 @@
...
-		if ($menu_category != "All" && $index['categories'][0] != $menu_category && !($menu_category == "Other" && !in_array($index['categories'][0], $visible_categories))) {
-			continue;
-		}
+<?php foreach ($pkg_info as $index):
+	if (isset($index['installed'])) {
+		continue;
+	}
 
+	if ($menu_category != "All" && $index['categories'][0] != $menu_category &&
+	  !($menu_category == "Other" && !in_array($index['categories'][0], $visible_categories))) {
```
Comment (sbeaver-netgate) [inline src/usr/local/www/pkg_mgr.php]: Style guide requires 4 space indent after a compound "if"

Comment (BBcan177) [inline src/usr/local/www/pkg_mgr.php]: np.. Just look ugly that way :)

File: src/usr/local/www/pkg_mgr.php [hunk 2]
```diff
@@ -130,12 +128,12 @@
 //		display_top_tabs($tab_array);
 }
 
-if (!$pkg_info || !is_array($pkg_info)):
-?>
+if (!$pkg_info || !is_array($pkg_info)):?>
```
Comment (sbeaver-netgate) [inline src/usr/local/www/pkg_mgr.php]: I have yet to find a text editor that understands PHP "alternate" control format :)

Comment (BBcan177) [inline src/usr/local/www/pkg_mgr.php]: What would you like that line to look like then? Drop the ?> to the following line?

Comment (sbeaver-netgate) [inline src/usr/local/www/pkg_mgr.php]: NO. Its fine the way you have it. I was just whining :)

**Pull request merged**